### ISSUE TITLE
Terraform output is broken in 0.7.4.  Can't pull the IP address information.

### ISSUE THREAD

**Issue by southsidedean** [dmacvicar/terraform-provider-libvirt#1037]
### Linux distribution

Ubuntu 22.04

### Terraform version

I've rolled back to 1.5.7 to test 0.7.4 against 0.7.0/0.7.1, due to the signing key issues in the older versions with terraform 1.6.0/1.6.1.

```sh
Terraform v1.5.7
on linux_amd64
```

### Provider and libvirt versions

provider = 0.7.4

Package: libvirt-daemon
Version: 8.0.0-1ubuntu7.7

___

## Checklist

- [ ] Is your issue/contribution related with enabling some setting/option exposed by libvirt that the plugin does not yet support, or requires changing/extending the provider terraform schema?

  - [ ] Make sure you explain why this option is important to you, why it should be important to everyone. Describe your use-case with detail and provide examples where possible.
  - [ ] If it is a very special case, consider using the _XSLT_ support in the provider to tweak the definition instead of opening an issue
  - [ ] Maintainers do not have expertise in every libvirt setting, so please, describe the feature and how it is used. Link to the appropriate documentation

- [X] Is it a bug or something that does not work as expected? Please make sure you fill the version information below:

## Description of Issue/Question

Terraform output is broken in 0.7.4.  Can't pull the IP address information.  I've double-checked that the qemu-agent is installed and running, etc.

Output from a 0.7.4 run yields:

```
tdean@monolith:~/tf-temp/cka-d-cluster-builder-lab$ terraform apply cka-plan.plan
module.worker.libvirt_volume.base-volume-qcow2[0]: Creating...
module.controlplane.libvirt_volume.base-volume-qcow2[0]: Creating...
module.worker.libvirt_cloudinit_disk.commoninit[0]: Creating...
module.worker.libvirt_cloudinit_disk.commoninit[1]: Creating...
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Creating...
module.controlplane.libvirt_volume.base-volume-qcow2[0]: Still creating... [10s elapsed]
module.worker.libvirt_volume.base-volume-qcow2[0]: Still creating... [10s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Still creating... [10s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Still creating... [10s elapsed]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Still creating... [10s elapsed]
module.controlplane.libvirt_volume.base-volume-qcow2[0]: Still creating... [20s elapsed]
module.worker.libvirt_volume.base-volume-qcow2[0]: Still creating... [20s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Still creating... [20s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Still creating... [20s elapsed]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Still creating... [20s elapsed]
module.controlplane.libvirt_volume.base-volume-qcow2[0]: Creation complete after 24s [id=/media/virtual-machines/control-plane--base.qcow2]
module.controlplane.libvirt_volume.volume-qcow2[0]: Creating...
module.worker.libvirt_volume.base-volume-qcow2[0]: Still creating... [30s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Still creating... [30s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Still creating... [30s elapsed]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Still creating... [30s elapsed]
module.controlplane.libvirt_volume.volume-qcow2[0]: Still creating... [10s elapsed]
module.worker.libvirt_volume.base-volume-qcow2[0]: Still creating... [40s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Still creating... [40s elapsed]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Still creating... [40s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Still creating... [40s elapsed]
module.controlplane.libvirt_volume.volume-qcow2[0]: Still creating... [20s elapsed]
module.worker.libvirt_volume.base-volume-qcow2[0]: Creation complete after 47s [id=/media/virtual-machines/worker-node--base.qcow2]
module.worker.libvirt_volume.volume-qcow2[1]: Creating...
module.worker.libvirt_volume.volume-qcow2[0]: Creating...
module.worker.libvirt_cloudinit_disk.commoninit[0]: Creation complete after 47s [id=/media/virtual-machines/worker-node-_init01.iso;c135ed37-ef2e-4626-93c2-517049927c02]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Creation complete after 47s [id=/media/virtual-machines/control-plane-_init01.iso;ca3e7717-1c3c-4b99-a8c0-353348ef310f]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Creation complete after 47s [id=/media/virtual-machines/worker-node-_init02.iso;ca9cfaf3-0097-4502-9f17-02dbcdf26cca]
module.controlplane.libvirt_volume.volume-qcow2[0]: Creation complete after 22s [id=/media/virtual-machines/control-plane-01.qcow2]
module.controlplane.libvirt_domain.virt-machine[0]: Creating...
module.worker.libvirt_volume.volume-qcow2[1]: Creation complete after 0s [id=/media/virtual-machines/worker-node-02.qcow2]
module.worker.libvirt_volume.volume-qcow2[0]: Creation complete after 0s [id=/media/virtual-machines/worker-node-01.qcow2]
module.worker.libvirt_domain.virt-machine[0]: Creating...
module.worker.libvirt_domain.virt-machine[1]: Creating...
╷
│ Error: couldn't retrieve IP address of domain id: a7983281-7f04-477d-8bfa-e9812922a490. Please check following: 
│ 1) is the domain running proplerly? 
│ 2) has the network interface an IP address? 
│ 3) Networking issues on your libvirt setup? 
│  4) is DHCP enabled on this Domain's network? 
│ 5) if you use bridge network, the domain should have the pkg qemu-agent installed 
│ IMPORTANT: This error is not a terraform libvirt-provider error, but an error caused by your KVM/libvirt infrastructure configuration/setup 
│  error retrieving interface addresses: error retrieving interface addresses: Guest agent is not responding: QEMU guest agent is not connected
│ 
│   with module.controlplane.libvirt_domain.virt-machine[0],
│   on .terraform/modules/controlplane/main.tf line 11, in resource "libvirt_domain" "virt-machine":
│   11: resource "libvirt_domain" "virt-machine" {
│ 
╵
╷
│ Error: couldn't retrieve IP address of domain id: 029a923b-85ad-4595-ae73-9e13b9b4de92. Please check following: 
│ 1) is the domain running proplerly? 
│ 2) has the network interface an IP address? 
│ 3) Networking issues on your libvirt setup? 
│  4) is DHCP enabled on this Domain's network? 
│ 5) if you use bridge network, the domain should have the pkg qemu-agent installed 
│ IMPORTANT: This error is not a terraform libvirt-provider error, but an error caused by your KVM/libvirt infrastructure configuration/setup 
│  error retrieving interface addresses: error retrieving interface addresses: Guest agent is not responding: QEMU guest agent is not connected
│ 
│   with module.worker.libvirt_domain.virt-machine[0],
│   on .terraform/modules/worker/main.tf line 11, in resource "libvirt_domain" "virt-machine":
│   11: resource "libvirt_domain" "virt-machine" {
│ 
╵
╷
│ Error: couldn't retrieve IP address of domain id: e3f29889-7147-47c8-b211-f8322c0a33bc. Please check following: 
│ 1) is the domain running proplerly? 
│ 2) has the network interface an IP address? 
│ 3) Networking issues on your libvirt setup? 
│  4) is DHCP enabled on this Domain's network? 
│ 5) if you use bridge network, the domain should have the pkg qemu-agent installed 
│ IMPORTANT: This error is not a terraform libvirt-provider error, but an error caused by your KVM/libvirt infrastructure configuration/setup 
│  error retrieving interface addresses: error retrieving interface addresses: Guest agent is not responding: QEMU guest agent is not connected
│ 
│   with module.worker.libvirt_domain.virt-machine[1],
│   on .terraform/modules/worker/main.tf line 11, in resource "libvirt_domain" "virt-machine":
│   11: resource "libvirt_domain" "virt-machine" {
│ 

```

The objects appear to be created without issue with 0.7.4, other than the outputs being broken.

Rolling back by setting the provider version to 0.7.0/0.7.1 with terraform 1.5.7 works fine:

```
tdean@monolith:~/tf-temp/cka-d-cluster-builder-lab$ terraform apply cka-plan.plan
module.controlplane.libvirt_volume.base-volume-qcow2[0]: Creating...
module.worker.libvirt_volume.base-volume-qcow2[0]: Creating...
module.worker.libvirt_cloudinit_disk.commoninit[0]: Creating...
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Creating...
module.worker.libvirt_cloudinit_disk.commoninit[1]: Creating...
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Still creating... [10s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Still creating... [10s elapsed]
module.controlplane.libvirt_volume.base-volume-qcow2[0]: Still creating... [10s elapsed]
module.worker.libvirt_volume.base-volume-qcow2[0]: Still creating... [10s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Still creating... [10s elapsed]
module.controlplane.libvirt_volume.base-volume-qcow2[0]: Still creating... [20s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Still creating... [20s elapsed]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Still creating... [20s elapsed]
module.worker.libvirt_volume.base-volume-qcow2[0]: Still creating... [20s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Still creating... [20s elapsed]
module.controlplane.libvirt_volume.base-volume-qcow2[0]: Creation complete after 24s [id=/media/virtual-machines/control-plane--base.qcow2]
module.controlplane.libvirt_volume.volume-qcow2[0]: Creating...
module.worker.libvirt_volume.base-volume-qcow2[0]: Still creating... [30s elapsed]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Still creating... [30s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Still creating... [30s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Still creating... [30s elapsed]
module.controlplane.libvirt_volume.volume-qcow2[0]: Still creating... [10s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Still creating... [40s elapsed]
module.worker.libvirt_volume.base-volume-qcow2[0]: Still creating... [40s elapsed]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Still creating... [40s elapsed]
module.worker.libvirt_cloudinit_disk.commoninit[1]: Still creating... [40s elapsed]
module.controlplane.libvirt_volume.volume-qcow2[0]: Still creating... [20s elapsed]
module.worker.libvirt_volume.base-volume-qcow2[0]: Creation complete after 49s [id=/media/virtual-machines/worker-node--base.qcow2]
module.worker.libvirt_volume.volume-qcow2[1]: Creating...
module.worker.libvirt_volume.volume-qcow2[0]: Creating...
module.worker.libvirt_cloudinit_disk.commoninit[1]: Creation complete after 49s [id=/media/virtual-machines/worker-node-_init02.iso;e3f0f52a-3ced-4ce3-94ef-1c03ef22ae7b]
module.controlplane.libvirt_cloudinit_disk.commoninit[0]: Creation complete after 49s [id=/media/virtual-machines/control-plane-_init01.iso;3b763408-791d-4095-b2a9-9d930b73674b]
module.worker.libvirt_cloudinit_disk.commoninit[0]: Creation complete after 49s [id=/media/virtual-machines/worker-node-_init01.iso;f7f9e4ad-b4c6-42ba-b3dd-1ccade030f60]
module.controlplane.libvirt_volume.volume-qcow2[0]: Creation complete after 25s [id=/media/virtual-machines/control-plane-01.qcow2]
module.controlplane.libvirt_domain.virt-machine[0]: Creating...
module.worker.libvirt_volume.volume-qcow2[1]: Creation complete after 1s [id=/media/virtual-machines/worker-node-02.qcow2]
module.worker.libvirt_volume.volume-qcow2[0]: Creation complete after 1s [id=/media/virtual-machines/worker-node-01.qcow2]
module.worker.libvirt_domain.virt-machine[1]: Creating...
module.worker.libvirt_domain.virt-machine[0]: Creating...
module.controlplane.libvirt_domain.virt-machine[0]: Still creating... [10s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Still creating... [10s elapsed]
module.worker.libvirt_domain.virt-machine[0]: Still creating... [10s elapsed]
module.controlplane.libvirt_domain.virt-machine[0]: Still creating... [20s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Still creating... [20s elapsed]
module.worker.libvirt_domain.virt-machine[0]: Still creating... [20s elapsed]
module.controlplane.libvirt_domain.virt-machine[0]: Still creating... [30s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Still creating... [30s elapsed]
module.worker.libvirt_domain.virt-machine[0]: Still creating... [30s elapsed]
module.controlplane.libvirt_domain.virt-machine[0]: Still creating... [40s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Still creating... [40s elapsed]
module.worker.libvirt_domain.virt-machine[0]: Still creating... [40s elapsed]
module.controlplane.libvirt_domain.virt-machine[0]: Still creating... [50s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Still creating... [50s elapsed]
module.worker.libvirt_domain.virt-machine[0]: Still creating... [50s elapsed]
module.controlplane.libvirt_domain.virt-machine[0]: Still creating... [1m1s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Still creating... [1m0s elapsed]
module.worker.libvirt_domain.virt-machine[0]: Still creating... [1m0s elapsed]
module.controlplane.libvirt_domain.virt-machine[0]: Still creating... [1m11s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Still creating... [1m10s elapsed]
module.worker.libvirt_domain.virt-machine[0]: Still creating... [1m10s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Provisioning with 'remote-exec'...
module.worker.libvirt_domain.virt-machine[1] (remote-exec): Connecting to remote host via SSH...
module.worker.libvirt_domain.virt-machine[1] (remote-exec):   Host: 10.0.1.184
module.worker.libvirt_domain.virt-machine[1] (remote-exec):   User: ubuntu
module.worker.libvirt_domain.virt-machine[1] (remote-exec):   Password: false
module.worker.libvirt_domain.virt-machine[1] (remote-exec):   Private key: true
module.worker.libvirt_domain.virt-machine[1] (remote-exec):   Certificate: false
module.worker.libvirt_domain.virt-machine[1] (remote-exec):   SSH Agent: false
module.worker.libvirt_domain.virt-machine[1] (remote-exec):   Checking Host Key: false
module.worker.libvirt_domain.virt-machine[1] (remote-exec):   Target Platform: unix
module.controlplane.libvirt_domain.virt-machine[0]: Provisioning with 'remote-exec'...
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec): Connecting to remote host via SSH...
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec):   Host: 10.0.1.160
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec):   User: ubuntu
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec):   Password: false
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec):   Private key: true
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec):   Certificate: false
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec):   SSH Agent: false
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec):   Checking Host Key: false
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec):   Target Platform: unix
module.worker.libvirt_domain.virt-machine[0]: Provisioning with 'remote-exec'...
module.worker.libvirt_domain.virt-machine[0] (remote-exec): Connecting to remote host via SSH...
module.worker.libvirt_domain.virt-machine[0] (remote-exec):   Host: 10.0.1.95
module.worker.libvirt_domain.virt-machine[0] (remote-exec):   User: ubuntu
module.worker.libvirt_domain.virt-machine[0] (remote-exec):   Password: false
module.worker.libvirt_domain.virt-machine[0] (remote-exec):   Private key: true
module.worker.libvirt_domain.virt-machine[0] (remote-exec):   Certificate: false
module.worker.libvirt_domain.virt-machine[0] (remote-exec):   SSH Agent: false
module.worker.libvirt_domain.virt-machine[0] (remote-exec):   Checking Host Key: false
module.worker.libvirt_domain.virt-machine[0] (remote-exec):   Target Platform: unix
module.worker.libvirt_domain.virt-machine[1] (remote-exec): Connected!
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec): Connected!
module.worker.libvirt_domain.virt-machine[0] (remote-exec): Connected!
module.controlplane.libvirt_domain.virt-machine[0]: Still creating... [1m21s elapsed]
module.worker.libvirt_domain.virt-machine[1]: Still creating... [1m20s elapsed]
module.worker.libvirt_domain.virt-machine[0]: Still creating... [1m20s elapsed]
module.worker.libvirt_domain.virt-machine[1] (remote-exec): Virtual Machine worker-node-02 is UP!
module.worker.libvirt_domain.virt-machine[1] (remote-exec): Wed Oct 11 15:10:47 UTC 2023
module.worker.libvirt_domain.virt-machine[0] (remote-exec): Virtual Machine worker-node-01 is UP!
module.worker.libvirt_domain.virt-machine[0] (remote-exec): Wed Oct 11 15:10:47 UTC 2023
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec): Virtual Machine control-plane-01 is UP!
module.worker.libvirt_domain.virt-machine[1]: Creation complete after 1m21s [id=32f3ef5a-af61-48a0-b5a5-de7a396e4a3d]
module.controlplane.libvirt_domain.virt-machine[0] (remote-exec): Wed Oct 11 15:10:47 UTC 2023
module.worker.libvirt_domain.virt-machine[0]: Creation complete after 1m21s [id=e44ccc8d-3aa5-4e66-a140-ae3fecf08842]
module.controlplane.libvirt_domain.virt-machine[0]: Creation complete after 1m22s [id=74f1d6f0-c0e3-4d07-9a9a-b70ef9195d34]

Apply complete! Resources: 11 added, 0 changed, 0 destroyed.

Outputs:

control-planes = {
  "ip_address" = [
    "10.0.1.160",
  ]
  "name" = [
    "control-plane-01",
  ]
}
worker-nodes = {
  "ip_address" = [
    "10.0.1.95",
    "10.0.1.184",
  ]
  "name" = [
    "worker-node-01",
    "worker-node-02",
  ]
}

```

### Setup

The terraform code is from my [GitHub repository](https://github.com/southsidedean/cka-d-cluster-builder-lab)

```
# Terraform code to stand up infrastructure to build
# an Open Source Kubernetes cluster
#
# Tom Dean
# tom@dean33.com
#
# Last edit 10/11/2023
#
# Based on the Terraform module for KVM/Libvirt Virtual Machine
# https://registry.terraform.io/modules/MonolithProjects/vm/libvirt/1.10.0
# Utilizes the dmacvicar/libvirt Terraform provider

# Let's set some variables!

# Cluster sizing: minimum one of each!
# We can set the number of control plane and worker nodes here

variable "control_plane_nodes" {
  type = number
  default = 1
}

variable "worker_nodes" {
  type = number
  default = 2
}

# Hostname prefixes
# This controls how the hostnames are generated

variable "cp_prefix" {
  type = string
  default = "control-plane-"
}

variable "worker_prefix" {
  type = string
  default = "worker-node-"
}

# Node sizing
# Start with the control planes

variable "cp_cpu" {
  type = number
  default = 2
}

variable "cp_disk" {
  type = number
  default = 25
}

variable "cp_memory" {
  type = number
  default = 8192
}

# On to the worker nodes

variable "worker_cpu" {
  type = number
  default = 2
}

variable "worker_disk" {
  type = number
  default = 25
}

variable "worker_memory" {
  type = number
  default = 8192
}

# Disk Pool to use
# Control Plane

variable "cp_diskpool" {
  type = string
  default = "default"
}

# Worker Nodes

variable "worker_diskpool" {
  type = string
  default = "default"
}

# User / Key information
# Same across all nodes, customize if you wish

variable "privateuser" {
  type = string
  default = "ubuntu"
}

variable "privatekey" {
  type = string
  default = "~/.ssh/id_ed25519"
}

variable "pubkey" {
  type = string
  default = "~/.ssh/id_ed25519.pub"
}

# Other node configuration

variable "timezone" {
  type = string
  default = "CST"
}

variable "osimg" {
  type = string
  default = "https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img"
}

# Set our Terraform provider here
# We're going to use libvirt on our local machine

terraform {
  required_providers {
    libvirt = {
      source = "dmacvicar/libvirt"
    }
  }
}

provider "libvirt" {
  uri = "qemu:///system"
}

# Module for building our control plane nodes

module "controlplane" {
  source  = "MonolithProjects/vm/libvirt"
  version = "1.10.0"

  vm_hostname_prefix = var.cp_prefix
  vm_count    = var.control_plane_nodes
  memory      = var.cp_memory
  vcpu        = var.cp_cpu
  pool        = var.cp_diskpool
  system_volume = var.cp_disk
  dhcp        = true
  ssh_admin   = var.privateuser
  ssh_private_key = var.privatekey
  ssh_keys    = [
    file(var.pubkey),
    ]
  time_zone   = var.timezone
  os_img_url  = var.osimg
}

# Module for building our worker nodes

module "worker" {
  source  = "MonolithProjects/vm/libvirt"
  version = "1.10.0"

  vm_hostname_prefix = var.worker_prefix
  vm_count    = var.worker_nodes
  memory      = var.worker_memory
  vcpu        = var.worker_cpu
  pool        = var.worker_diskpool
  system_volume = var.worker_disk
  dhcp        = true
  ssh_admin   = var.privateuser
  ssh_private_key = var.privatekey
  ssh_keys    = [
    file(var.pubkey),
    ]
  time_zone   = var.timezone
  os_img_url  = var.osimg
}

# Outputs

output "control-planes" {
  value = module.controlplane
}

output "worker-nodes" {
  value = module.worker
}

```

### Steps to Reproduce Issue

```
terraform init
terraform plan -out cka-plan.plan
terraform apply cka-plan.plan
```
___

## Additional information:

Do you have SELinux or Apparmor/Firewall enabled? Some special configuration?

No

Have you tried to reproduce the issue without them enabled?

No, the issue goes away with older provider versions on the same system.

**Comment by rgl** [comment]
maybe, is this because of this timeouts change?

https://github.com/dmacvicar/terraform-provider-libvirt/commit/01e507d95b8b49fb6d94b8ecde7e34ee4cb1c567#diff-ec1bf6b27daed6653f7d487358b1b3b311e60427f79397bc63e6c74eecc3667eL70-R72

**Comment by southsidedean** [comment]
Thanks for the update!

**Comment by momoah** [comment]
Hello,
Having the same issue with my bridge interface (no problem with the virtual bridge created by libvirt). Does this mean we need to wait for a 0.7.5 or can we just build it from source?

**Comment by thequailman** [comment]
You can work around this by setting the version of the plugin to `0.7.1`.

**Comment by momoah** [comment]
> You can work around this by setting the version of the plugin to `0.7.1`.

Superb! That worked a treat!
I'll stick with this version until this is fixed in later versions.

**Comment by kmasaryk** [comment]
Setting the version to `0.7.1`  appears to fix the immediate problem with QEMU agent communication since Terraform isn't throwing an error now when `qemu_agent = true` but it's still unable to pull the IP for me. Appears to be due to the issue described in #1047 which is potentially fixed in the PR #1048. 

Performing a `terraform destroy` does produce valid addresses in the output so the communication is there but looks like `wait_for_lease = true` isn't actually waiting during deployment.
